sycl/test-e2e/Matrix/joint_matrix_out_bounds_impl.hpp

```diff
@@ -0,0 +1,153 @@
+#include <iostream>
+#include <random>
+#include <sycl/sycl.hpp>
+
+using namespace sycl;
+using namespace sycl::ext::oneapi::experimental::matrix;
+using bfloat16 = sycl::ext::oneapi::bfloat16;
+
+#define TM 8
+#define TN SG_SZ
```

Comment:
In general case TN can be different from SG_SZ. Could you please define TN in .cpp file similar to SG_SZ?